packages/fluentui/docs/src/examples/components/Form/Usage/FormExampleErrorAndSuccessful.shorthand.tsx

```diff
@@ -1,32 +0,0 @@
-import * as React from 'react';
```

Comment:
Is there a non-shorthand example for error and successful ?

Reply:
Yes `FormExampleErrorAndSuccessful.tsx`